iron-collapse.html

```diff
@@ -161,38 +192,26 @@
     },
 
     _openedChanged: function() {
+      this.setAttribute('aria-expanded', this.opened);
+      this.setAttribute('aria-hidden', !this.opened);
       if (this.opened) {
-        this.setAttribute('aria-expanded', 'true');
-        this.setAttribute('aria-hidden', 'false');
-
         this.toggleClass('iron-collapse-closed', false);
```

Comment:
tbh I think you can move this to be 
`this.toggleClass(this.opened ? 'iron-collapse-closed' 'iron-collapse-opened' : , false);`